js/app.js

```diff
@@ -4,6 +4,51 @@ $(document).ready( function() {
   // check to make sure JS is loaded
   console.log('JS is loaded!');
 
-  // your code here
+  
 
-});
+  var displaySongInfo = function(data){
+    var $results = $("#results");
+    $results.empty();
+    console.log(data.tracks.items.length);
+    if (data.tracks.items.length === 0){$results.append("<h1>That search was no good try another.</h1>")}
+
+    data.tracks.items.forEach(function(element){
```

Comment:
Hey Jonathan,

Code looks great - I'm glad you're feeling comfortable with it. Don't worry too much about the CSS - there will be plenty more opportunities to work on it in with other projects we do. 

One note I do have for this code is when you're using html in template literals it really helps with the readability of your code to make new lines for each line of html. For instance lines 17-19 could look like this: 
```
$results.append(`
          <div class="col-sm-4 row-eq-height">
            <img src=${element.album.images[2].url} alt="images/spotify_2015.png"><br>
            <p>${element.name} by ${element.album.artists[0].name}</p><br>
            <a class="btn btn-outline-success" href=${element.preview_url}>Preview</a>
         </div>
`)

```

This is way easier to debug when you see the code like this. 

Great job!
Matt